### ISSUE TITLE
When to run eval?

### ISSUE THREAD

**Issue by gngdb** [BayesWatch/tf-variational-dropout#5]
It's annoying that the eval script is asynchronous to the training script for the CIFAR-10 model. To run it on the same GPU we'd have to stop the current training, clear the graph, open the model again, restart the graph, make a new session and then run all the test examples through. Then restart the training again.

It might be possible to run the evaluation in the same session, but I'm not totally sure with this weird MonitoredTrainingSession.

Maybe it's faster to open and close sessions in the `cifar_train` script, then iterate over a validation set in a new session? Then at least we don't have to rebuild computational graphs and go back and forth to the disk.
